.github/workflows/stale.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      - uses: actions/stale@6f05e4244c9a0b2ed3401882b05d701dd0a7289b # v7.0.0
```

Comment:
Compared with https://github.com/nextcloud/android-library/pull/1029/files
- 6f05e4244c9a0b2ed3401882b05d701dd0a7289b is v5
- 5ebf00ea0e4c1561e9b43a292ed34424fb1d4578 is v6